### PR TITLE
Fix readme doc issues of jca

### PR DESCRIPTION
**Pull request by backwind1233**
As title.

### CODE REVIEW

**Comment by backwind1233** [comment]
Link with https://github.com/Azure/azure-sdk-for-java/issues/27329, finding doc errors why addressing the issue.

**Comment by backwind1233** [comment]
@chenrujun please help review this PR.

File: keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-client-side/README.md [hunk 1]
```diff
@@ -15,7 +15,9 @@ This sample should work together with [azure-spring-boot-sample-keyvault-certifi
    ```
    source script/setup.sh
    ```
-2. Option 2 - If you created the resource via the Azure Portal, you need configure the application.yml manually, please replace the placeholders with the resources you created.   
+2. Option 2 - If you created the resource via the Azure Portal, you need configure the application.yml manually, please replace the placeholders with the resources you created.
+   > Attention: The service principal must be configured with get and list permissions of certificate, key, secret.
```
Comment (chenrujun) [inline keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-client-side/README.md:19]:
```
key: get
secret: get
certificate: get, list
```
For key and secret, only `get` permission is required.

File: keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-server-side/README.md [hunk 2]
```diff
@@ -49,7 +49,8 @@ You can also create resources manually via Azure Portal. Please follow:
    - If you cannot use managed identity, you can register your application with AAD, see [register app with AAD][register_app_with_AAD]. The registration also creates a second application object that identifies your app.
 2. Create the key vault and certificates. Please refer to [create key vault and certificates][create_key_vault_and_certificates]
 3. Make the key vault accessible to your service principal. Please refer to [assign key vault access policy][assign_key_vault_access_policy]
-4. You need manually configure the application.yml, replace the placeholders with the resources you created in the Azure Portal.
+   > Attention: The service principal must be configured with get and list permissions of certificate, key, secret.
```
Comment (chenrujun) [inline keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-server-side/README.md:52]: Same here.

**Review by chenrujun — CHANGES_REQUESTED** [review]

File: keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-server-side/README.md (hunk 2 above)
Comment (backwind1233) [reply keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-server-side/README.md]: updated

File: keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-client-side/README.md [hunk 3]
```diff
@@ -15,7 +15,12 @@ This sample should work together with [azure-spring-boot-sample-keyvault-certifi
    ```
    source script/setup.sh
    ```
-2. Option 2 - If you created the resource via the Azure Portal, you need configure the application.yml manually, please replace the placeholders with the resources you created.   
+2. Option 2 - If you created the resource via the Azure Portal, you need configure the application.yml manually, please replace the placeholders with the resources you created.
+   > Attention: The service principal must be configured with permissions:   
+   > Certificate Permissions: configure with **get and list** permissions.  
+   > Key Permissions: configure with get permission.
```
Comment (chenrujun) [inline keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-client-side/README.md:21]: Nit: `get` -> `**get**`.

File: keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-client-side/README.md [hunk 4]
```diff
@@ -15,7 +15,12 @@ This sample should work together with [azure-spring-boot-sample-keyvault-certifi
    ```
    source script/setup.sh
    ```
-2. Option 2 - If you created the resource via the Azure Portal, you need configure the application.yml manually, please replace the placeholders with the resources you created.   
+2. Option 2 - If you created the resource via the Azure Portal, you need configure the application.yml manually, please replace the placeholders with the resources you created.
+   > Attention: The service principal must be configured with permissions:   
+   > Certificate Permissions: configure with **get and list** permissions.  
+   > Key Permissions: configure with get permission.  
+   > Secret Permissions: configure with get permission.
```
Comment (chenrujun) [inline keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-client-side/README.md:22]: Same here.

File: keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-server-side/README.md [hunk 5]
```diff
@@ -49,7 +49,11 @@ You can also create resources manually via Azure Portal. Please follow:
    - If you cannot use managed identity, you can register your application with AAD, see [register app with AAD][register_app_with_AAD]. The registration also creates a second application object that identifies your app.
 2. Create the key vault and certificates. Please refer to [create key vault and certificates][create_key_vault_and_certificates]
 3. Make the key vault accessible to your service principal. Please refer to [assign key vault access policy][assign_key_vault_access_policy]
-4. You need manually configure the application.yml, replace the placeholders with the resources you created in the Azure Portal.
+   > Attention: The service principal must be configured with permissions:   
+   > Certificate Permissions: configure with **get and list** permissions.  
+   > Key Permissions: configure with get permission.
```
Comment (chenrujun) [inline keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-server-side/README.md:54]: Same here.

File: keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-server-side/README.md [hunk 6]
```diff
@@ -49,7 +49,11 @@ You can also create resources manually via Azure Portal. Please follow:
    - If you cannot use managed identity, you can register your application with AAD, see [register app with AAD][register_app_with_AAD]. The registration also creates a second application object that identifies your app.
 2. Create the key vault and certificates. Please refer to [create key vault and certificates][create_key_vault_and_certificates]
 3. Make the key vault accessible to your service principal. Please refer to [assign key vault access policy][assign_key_vault_access_policy]
-4. You need manually configure the application.yml, replace the placeholders with the resources you created in the Azure Portal.
+   > Attention: The service principal must be configured with permissions:   
+   > Certificate Permissions: configure with **get and list** permissions.  
+   > Key Permissions: configure with get permission.  
+   > Secret Permissions: configure with get permission.
```
Comment (chenrujun) [inline keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-server-side/README.md:55]: Here.

**Review by chenrujun — APPROVED** [review]

File: keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-server-side/README.md (hunk 6 above)
Comment (backwind1233) [reply keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-server-side/README.md:55]: all updated

**Review by chenrujun — COMMENTED** [review]
LGTM.

**Pull request merged**